resources/views/filament/pages/audit-log.blade.php

```diff
@@ -48,56 +48,65 @@
-                                    ->whereIn('id', collect($newValue)->pluck('id')->filter())
-                                    ->get();
-                            }
+
+                        @foreach (data_get($changes, 'attributes', []) as $field => $change)
+                            @php
+                                $oldValue = data_get($changes, "old.{$field}");
+                                $newValue = data_get($changes, "attributes.{$field}");
+                                $isRelationship = method_exists($this->record, $field);
+                                $newRelatedModels = collect();
+                                $oldRelatedModels = collect();
+                                if ($isRelationship) {
+                                    $oldRelatedModels = $this->record->{$field}()->getRelated()
+                                        ->whereIn('id', collect($oldValue)->pluck('id')->filter())
+                                        ->get();
+                                    $newRelatedModels = $this->record->{$field}()->getRelated()
+                                        ->whereIn('id', collect($newValue)->pluck('id')->filter())
+                                        ->get();
+                                }
                             @endphp
+
                             <x-filament-tables::row @class(['bg-gray-100/30' => $loop->even])>
                                 <x-filament-tables::cell width="15%" class="px-4 py-2 align-top sm:first-of-type:ps-6 sm:last-of-type:pe-6">
                                     {{ $this->getFieldLabel($field) }}
                                 </x-filament-tables::cell>
+
                                 <x-filament-tables::cell width="40%" class="px-4 py-2 align-top break-all !whitespace-normal">
-                                    @if($oldRelatedModels->isNotEmpty())
-                                        @foreach($oldRelatedModels as $relatedModel)
+                                    @if ($oldRelatedModels->isNotEmpty())
+                                        @foreach ($oldRelatedModels as $relatedModel)
                                             <div class="inline-block">
                                                 <x-filament::badge>
                                                     {{ $relatedModel->name }}
                                                 </x-filament::badge>
                                                 {{ collect($oldValue)->where('id', $relatedModel->name)->get('attributes') }}
                                             </div>
                                         @endforeach
-                                    @elseif(is_array($oldValue))
+                                    @elseif ($this->getIsImageField($field))
+                                        <img src="{{ $oldValue }}" alt="Old Image" class="max-w-full h-auto"/>
```

Comment:
This file's diff is mostly just whitespace changes. Lines 84-85 and 107-108 are the actual logic changes in the file.